.gitignore

```diff
@@ -131,3 +131,4 @@ newfits.fits
 eaConf.json
 exotic.log.*
 pl_names.json
+/.project
```

Comment:
What is the `.project` used by?

Reply:
Eclipse work file - never part of source, and adding it to the .gitignore keeps it from being checked in